### PR TITLE
Integrate pytest-benchmark, add selector and full_game tests.

### LINKED ISSUE #314: Look into integrating pytest-benchmark
https://pypi.python.org/pypi/pytest-benchmark

### PR DESCRIPTION
**Pull request by smallnamespace**
Closes #314

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 hearthstone
 pytest
+pytest-benchmark
```
Comment (jleclanche) [inline requirements.txt]: This should only be pip installed on travis.

File: pytest.ini [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --benchmark-skip
```
Comment (jleclanche) [inline pytest.ini]: Cant you just exclude the benchmark file or something?

File: requirements.txt (hunk 1 above)
Comment (smallnamespace) [inline requirements.txt]: Done

File: pytest.ini (hunk 2 above)
Comment (smallnamespace) [inline pytest.ini]: It was tricky, but for you I figured it out.

**Comment by smallnamespace** [comment]
@jleclanche 

Comments all addressed, good to push?

File: .travis.yml [hunk 3]
```diff
@@ -7,10 +7,13 @@ cache:
   directories:
     - $HOME/.pip-cache/
     - $HOME/virtualenv/python3.5
-script:
-  # Travis will automatically detect requirements.txt and run pip install
+install:
+  - pip install -r requirements.txt
```
Comment (jleclanche) [inline .travis.yml]: That part I believe isn't necessary.

Comment (smallnamespace) [inline .travis.yml]: It is; if you override install at all, Travis no longer defaults to `pip install -r ...`

**Comment by jleclanche** [comment]
gtm

**Pull request merged**